README.md

```diff
@@ -40,10 +40,11 @@ To learn more about the cloud-native architecture of ODC go to the [ODC document
 
 _Minimum system requirement per `outsystemscc` instance: 2 GB RAM, 2x 1GHz+ CPU._
 
-To install, use either the binary or Docker option. There are several advantages of running `outsystemscc` as a Docker image as opposed to a binary, if your system supports it:
+To install, use either the binary or Docker option. Run the binary on Linux, or use the Docker image on any OS that supports Docker. Running `outsystemscc` as a Docker image offers several advantages if your system supports it:
 
 * You always run the latest release. You don't need to reinstall each new release.
 * `outsystemscc` can run on Windows or any system that supports Docker.
+    * You can use the outsystemscc Linux binary within WSL on Windows, or use the outsystemscc Linux container with Docker fully installed within WSL2 on Windows.
```

Comment:
This seems to contradict the guidance:

> You can run the binary on any Windows version that supports [WSL2](https://docs.microsoft.com/en-us/windows/wsl/).

Can we pls ensure consistency? It's not clear to me what I can only do on WSL2 vs WSL.

Reply:
Pushed an update to maintain consistency